packages/calcite-components/src/components/action/action.tsx

```diff
@@ -298,10 +298,10 @@ export class Action
       buttonId,
       messages,
     } = this;
-    const labelFallback = label || text;
-    const ariaLabel = labelFallback
-      ? `${labelFallback}${indicator ? ` (${messages.indicator})` : ""}`
-      : "";
+    const labelFallback = label || text || "";
+    const ariaLabel = indicator
+      ? messages.indicatorLabel?.replace("{label}", labelFallback)
```

Comment:
Don't think we can start using new message string before translations are back. This will be remove `aria-label` for all non-english locales.

Ex: 
```
 <calcite-action icon="banana" scale="l" label="test" indicator text="test2" lang="bs"></calcite-action>
 ```
 
 ☝️  wouldn't add aria-label to button in shadowRoot.

Alternate: we can request a generate build which return english strings for all locales or wait until the translations are back before start using `indicatorLabel` string.

Reply:
Should we just add the string to the messages for now then and move this issue to another milestone? What would you recommend?

Reply:
>Should we just add the string to the messages for now then and move this issue to another milestone? What would you recommend?

Yup that works.

Second creating separate issue for strings.